client/lcd/swagger-ui/swagger.yaml

```diff
@@ -635,6 +635,23 @@ paths:
       description: Bech32 AccAddress of Delegator
```

Comment:
If you want to take a look on the swagger file copy and paste the raw file in https://editor.swagger.io/